src/containers/WelcomePage/components/LastUsedContent.tsx

```diff
@@ -39,18 +39,25 @@ const LastUsedContent: FC<Props & tType> = ({ articleId, locale, t }) => {
   return (
     <div {...classes('result')}>
       {article && (
-        <Link
-          {...classes('link')}
-          to={toEditArticle(article.id, article.articleType)}>
-          {article.title.title} ({t('article.lastUpdated')}{' '}
-          {article && formatDate(article.updated)})
-        </Link>
+        <Fragment>
```

Comment:
Trur du kan bytte ut fragment med tomme tags.
```suggestion
        <>
```

Reply:
tomme tags er en fragment, er bare short form.
Personlig foretrekker jeg eksplisitte fragments, men kan bytte om dere foretrekker short form

Reply:
Det er det vi har gjort mest av i kodebasen så det er fint om du gjør det likt.